From 8c4cf161a8b42749e986a3503f6cd4f3b5682fe3 Mon Sep 17 00:00:00 2001
From: Chen-Yu Tsai <wens@csie.org>
Date: Fri, 7 Dec 2018 16:47:19 +0800
Subject: [PATCH] dt-bindings: rtc: sun6i-rtc: Fix register range in example
Git-commit: 8c4cf161a8b42749e986a3503f6cd4f3b5682fe3
Patch-mainline: v5.0
References: bsc#1120902

The register range for the RTC extends beyond 0x54.
Use the size from the user manual's memory map instead.

Fixes: 9765d2d94309 ("rtc: sun6i: Add sun6i RTC driver")
Signed-off-by: Chen-Yu Tsai <wens@csie.org>
Signed-off-by: Alexandre Belloni <alexandre.belloni@bootlin.com>
Reviewed-by: Fabian Baumanis <fabian.baumanis@suse.com>
---
 Documentation/devicetree/bindings/rtc/sun6i-rtc.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/devicetree/bindings/rtc/sun6i-rtc.txt
+++ b/Documentation/devicetree/bindings/rtc/sun6i-rtc.txt
@@ -19,7 +19,7 @@ Example:
 
 rtc: rtc@01f00000 {
 	compatible = "allwinner,sun6i-a31-rtc";
-	reg = <0x01f00000 0x54>;
+	reg = <0x01f00000 0x400>;
 	interrupts = <0 40 4>, <0 41 4>;
 	clock-output-names = "osc32k";
 	clocks = <&ext_osc32k>;
